spec/schemas/_common.query_dsl.yaml

```
@@ -316,6 +316,16 @@ components:
           format: float
         _name:
           type: string
+    ValueType:
+      type: object
+      properties:
+        value_type:
+          description: |-
+            Specifies the types of values used for filtering. Valid values are default and bitmap. If omitted, the value defaults to default.
```

I think we support `default: default`? Either way either:

```suggestion
            Specifies the types of values used for filtering.
```

or

```suggestion
            Specifies the types of values used for filtering. Valid values are `default` and `bitmap`. Default is `default`.
```